packages/anywidget/src/widget.js

```diff
@@ -242,6 +242,35 @@ function throw_anywidget_error(source) {
 	throw source;
 }
 
+/**
+ * @template T
+ * @param {import("@anywidget/types").AnyModel} model
+ * @param {any} [action]
+ * @param {{ timeout?: number }} [options]
+ */
+export function dispatch(model, action, { timeout = 3000 } = {}) {
+	let id = Date.now().toString(36);
```

Comment:
I found that you really need a uuid here. In lonboard, the deck TileLayer can make multiple requests at the same time and this time-based ID isn't stable enough. Switching to `uuid`'s `v4()` fixed my issues.

Reply:
ok great to know. i will switch